### PR TITLE
New version: yt-dlp.yt-dlp.nightly version 2024.07.03.232825

### PR DESCRIPTION
**Pull request by Dvd-Znf**
Checklist for Pull Requests
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/161659)

### CODE REVIEW

File: manifests/y/yt-dlp/yt-dlp/nightly/2024.07.03.232825/yt-dlp.yt-dlp.nightly.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+  Networking changes
+  - Request Handler: requests: Bump minimum requests version to 2.32.2 (#10079) by bashonly
+
+  Misc. changes
+  - build
+    - Bump Pyinstaller to >=6.7.0 for all builds (#10069) by bashonly, seproDev
+    - Cache dependencies for macos job (#10088) by bashonly
+    - Use macos-12 image for yt-dlp_macos (#10063) by bashonly
+  - cleanup: Add more ruff rules (#10149) by seproDev
```
Comment (aisgbnok) [inline manifests/y/yt-dlp/yt-dlp/nightly/2024.07.03.232825/yt-dlp.yt-dlp.nightly.locale.en-US.yaml:78]: Update `ReleaseNotes`

```suggestion
ReleaseNotes: |-
  Core changes
  - Fix --ignore-no-formats-error (#10345) by Grub4K

  Extractor changes
  - douyutv: Do not use dangerous javascript source/URL (#10347) by LeSuisse
```

File: manifests/y/yt-dlp/yt-dlp/nightly/2024.07.03.232825/yt-dlp.yt-dlp.nightly.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ReleaseNotesUrl: https://github.com/yt-dlp/yt-dlp-nightly-builds/releases/tag/2024.06.24.232830
```
Comment (aisgbnok) [inline manifests/y/yt-dlp/yt-dlp/nightly/2024.07.03.232825/yt-dlp.yt-dlp.nightly.locale.en-US.yaml:79]: Update `ReleaseNotesUrl`

```suggestion
ReleaseNotesUrl: https://github.com/yt-dlp/yt-dlp-nightly-builds/releases/tag/2024.07.03.232825
```

**Review by aisgbnok — CHANGES_REQUESTED** [review]
Update `ReleaseNotes` and `ReleaseNotesUrl` to match [`2024.07.03.232825` release tag.](https://github.com/yt-dlp/yt-dlp-nightly-builds/releases/tag/2024.07.03.232825)

**Review by aisgbnok — APPROVED** [review]

**Review by stephengillie — APPROVED** [review]

**Comment by stephengillie** [comment]
- https://github.com/microsoft/winget-pkgs/issues/142854

**Pull request merged**